From d8131f3d995ab827412807d66b407e35d496c978 Mon Sep 17 00:00:00 2001
From: Yu Yue <yuyue@mprc.pku.edu.cn>
Date: Mon, 31 Oct 2011 11:29:04 +0800
Subject: [PATCH 047/641] UniCore64: Add function dump_fpu

Fixed error:
file-mmu.c:(.text+0x5aeec): undefined reference to `dump_fpu'

Signed-off-by: Yu Yue <yuyue@mprc.pku.edu.cn>
---
 arch/unicore64/kernel/process.c |   13 +++++++++++++
 1 file changed, 13 insertions(+)

diff --git a/arch/unicore64/kernel/process.c b/arch/unicore64/kernel/process.c
index eb596e4..4316735 100644
--- a/arch/unicore64/kernel/process.c
+++ b/arch/unicore64/kernel/process.c
@@ -1,4 +1,5 @@
 #include <linux/kernel.h>
+#include <linux/elfcore.h>
 
 /**
  * kernel_thread() - Create a kernel thread
@@ -95,3 +96,15 @@ void cpu_idle(void)
 	/* FIXME */
 	BUG();
 }
+
+/**
+ * dump_fpu() -
+ * @regs:
+ * @fp:
+ */
+int dump_fpu(struct pt_regs *regs, elf_fpregset_t *fp)
+{
+	/* FIXME: fill in the fpe structure for a core dump... */
+	BUG();
+	return 0;
+}
-- 
1.7.9.5

